File: 4.js
start = Date.now()
for (var i = 1; i < 1001; i++) {
    if (i % 5 == 0) {
        console.log(i ** 10)
    } else {
        console.log(i ** 123)
    }
}
end = Date.now() - start
console.log(end + " ms")

File: 4.py
from time import time
start = time()
for i in range(1, 1001):
    if i % 5 == 0:
        print(i ** 10)
    else:
        print(i ** 123)
end = round(time() - start, 3)
print(str(end) + " ms")